packages/medusa/src/api/store/return/middlewares.ts

```diff
@@ -1,15 +1,15 @@
-import { MiddlewareRoute } from "@medusajs/framework/http"
 import {
   validateAndTransformBody,
   validateAndTransformQuery,
 } from "@medusajs/framework"
+import { MiddlewareRoute } from "@medusajs/framework/http"
 import * as QueryConfig from "./query-config"
 import { ReturnsParams, StorePostReturnsReqSchema } from "./validators"
 
 export const storeReturnRoutesMiddlewares: MiddlewareRoute[] = [
   {
     method: ["POST"],
-    matcher: "/store/returns",
+    matcher: "/store/return",
```

Comment:
q: should the enpoint be `/store/returns`?

Reply:
@olivermrbl shall we change the route and call it a breaking change?

Reply:
Would it be fair to say it wasn't functional in the previous state, and label it as a fix rather than a breaking change?